config/config.php

```diff
@@ -229,4 +229,5 @@
     $services->set(\PHPStan\PhpDocParser\Lexer\Lexer::class);
     $services->set(TypeParser::class);
     $services->set(ConstExprParser::class);
+    $services->set(\Rector\Core\Util\FileHasher::class);
```

Comment:
This should be autoloaded with PSR-4 autodiscovery. What happens when you remove it?

Reply:
done